src/mcp/tools/crashlytics/get_issue_details.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { getIssueDetails } from "../../../crashlytics/getIssueDetails.js";
+
+export const get_issue_details = tool(
+  {
+    name: "get_issue_details",
+    description: "Gets the details about a specific crashlytics issue.",
+    inputSchema: z.object({
+      app_id: z
+        .string()
+        .describe(
+          "AppId for which the issues list should be fetched. For an Android application, read the mobilesdk_app_id value specified in the google-services.json file for the current package name. For an iOS Application, read the GOOGLE_APP_ID from GoogleService-Info.plist. If neither is available, use the `firebase_list_apps` tool to find an app_id to pass to this tool.",
+        ),
+      issue_id: z
+        .string()
+        .describe(
+          "The issue ID for which the details needs to be fetched. This is the value of the field `id` in the list of issues. Defaults to the first id in the list of issues.",
+        ),
+    }),
+    annotations: {
+      title: "Gets the details of a specific issue.",
+      readOnlyHint: true,
+    },
+    _meta: {
+      requiresAuth: true,
+      requiresProject: true,
```

Comment:
One thing I encountered as I was playing around with this is that for some of these analysis tools, you may want to run them from a different directory. Is it easier if we allow you to pass in a project id? It certainly makes it a bit harder to test.